docs/source/restore/index.rst

```diff
@@ -6,162 +6,79 @@ Restore
    :hidden:
    :maxdepth: 2
 
-   download-files-command
+   restore-tables
+   restore-schema
+   examples
 
-The following procedure lets you restore data to a new cluster to get a cluster clone from snapshots stored in a backup location.
+.. contents::
+   :depth: 2
+   :local:
```

Comment:
```suggestion
   :local:

The Restore procedure allows you to restore the data from a Scylla Manager Backup to a *new* ScyllaDB cluster.

The restore procedure has three steps:
- Downloading the SSTbales files from the backup location to each of the new cluster nodes (tzach: each of the nodes? few of the nodes?)
- Starting the Scylla process, which reads these SSTabkes and streams the parsed data to the correct nodes internally using the [Load and Stream feature](https://docs.scylladb.com/stable/operating-scylla/nodetool-commands/refresh.html#load-and-stream)
- Running a Repair process to sync the restore nodes.
```

Reply:
I'm confused about `restore-tables` and `restore-schema`
should one run one before the other?

Reply:
You can find information about the order below:
```
Restore types
=============

Restore task has to be one of two types:

  * :doc:`restore tables <restore-tables>` - restores the content of the tables (rows)

  * :doc:`restore schema <restore-schema>` - restores the ScyllaDB cluster schema

Each of those types has required prerequisites and follow-up actions.
For more information, please read given restore type documentation.

If both the schema and the content of the tables need to be restored, you must start with restoring the schema. Only after the schema is successfully restored can you proceed with restoring the content of the tables.

```

Reply:
The processes of restoring schema/tables are located in their own subsections.